update/update_2.3.5-2.4.php

```diff
@@ -124,10 +123,19 @@
 	// Add folders at the end of the files list to keep the order (files, folders)
 	$update['items'] = array_merge($update['items'], $folders);
 }
- 
+
 // check version:
-if(!in_array($settings['version'], $update['version'])) {
-	$update['errors'][] = 'Error in line '.__LINE__.': This update file doesn\'t work with the current version.';
+if(!file_exists('../config/VERSION')) {
```

Comment:
Nerf, the file VERSION reisdes in `forum/config/VERSION` and the update script resides in `forum/update/update_2.3.5-2.4.php`. When the update script is requested, it will be included in `forum/index.php`. The path must be given relative to `forum/index.php`. I'll correct it.